src/anm/analytics.js

```diff
@@ -16,27 +13,15 @@ var Analytics = function () {
             var data = JSON.stringify(self.queue);
             self.queue = [];
 
-            if (supportSendBeacon) {
+            if (navigator.sendBeacon) {
                 navigator.sendBeacon(animatronUrl, data);
             } else {
-                sendViaGif(data);
+                engine.ajax(animatronUrl, null, null, 'POST', null, data, false);
             }
         }
-        if (!supportSendBeacon) {
-            setTimeout(event, timeout);
-        }
     };
 
-    var sendViaGif = function (data) {
-        if (!beacon) {
-            beacon = engine.createStatImg();
```

Comment:
the problem is not that the data won't be sent, but that there will be extra request: OPTIONS. no analytics software sends xhr requests because of that, everybody does it via image to minimize number of calls